app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -491,4 +511,46 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
     public boolean isBackEnabled() {
         return false;
     }
-}
+    
+    private void populateAvailableAppsSpinner() {
+        Spinner spinner = (Spinner) findViewById(R.id.app_selection_spinner);
+        ArrayList<ApplicationRecord> readyApps = CommCareApplication._().getReadyAppRecords();

```

Comment:
Does getGlobalStorage(ApplicationRecord.class) return apps in a reliable order, or should they be sorted here? Not a big deal for most users, but I can see FMs having a dozen apps installed.